tiles/src/main/java/com/protomaps/basemap/layers/Places.java

```diff
@@ -287,12 +304,14 @@ public void processOsm(SourceFeature sf, FeatureCollector features) {
       //feat.setSortKey(minZoom * 1000 + 400 - populationRank * 200 + placeNumber.incrementAndGet());
       feat.setSortKey(getSortKey(minZoom, kindRank, populationRank, population, sf.getString("name")));
 
+      // This is only necessary when prepping for raster renderers
+      feat.setBufferPixels(16);
```

Comment:
If Protomaps is only targeting vector renderers than this could be set to 0 to reduce file size

Reply:
I think we still want a buffer for drawing townspots correctly very close to tile edges?

Reply:
If your label grid size isn't a factor of 256/power of 2 then you need a bit extra in order for label grid squares to be aware of points in the square from neighboring tiles otherwise the density might get off a bit around tile boundaries. Looks like 24 is the only problematic one. I think a buffer of 16 should be the minimum that works with label grid of 24.

If you want to go smaller you could either switch label grid to a power of 2 or use `FeatureMerge.removePointsOutsideBuffer` in `postProcess`

Reply:
I think the min buffer pixel limit should be `label grid size - gcd(256, label grid size)` - I should probably update planetiler to use that limit and post-filter to whatever you set here so users don't need to now or care about that implementation detail...